docs/authoring.md

```diff
@@ -86,78 +86,140 @@ To make your component available as a NuGet package, it is important to include
 
 This section describes the steps needed to consume a C#/WinRT component for the following scenarios:
 
-- [C++/WinRT desktop applications (Unpackaged)](#consuming-from-cwinrt-unpackaged)
-- [C# desktop applications](#consuming-from-c-applications) (.NET 5 or later)
-- [Packaged applications](#Consuming-from-packaged-applications)
-- [Out of process components](#Consuming-an-out-of-process-component)
+- [C# desktop applications](#consuming-from-c-applications) 
+- [C++ desktop applications](#consuming-from-c-applications-1)
+- [Windows App SDK applications](#windows-app-sdk-applications)
+- [Out of process components](#consuming-an-out-of-process-component)
 
-### Consuming from C++/WinRT (Unpackaged)
+### Consuming from C# applications
+
+Consuming a C#/WinRT component from C#/.NET apps is supported by both package reference or project reference.

```

Comment:
Perf offline discussion, might want to reword this as both lines seem to say the same.  Also since you are saying both, might want to use and rather than or.